wgpu-core/src/track/mod.rs

```diff
@@ -512,28 +512,52 @@ impl<A: HalApi> RenderBundleScope<A> {
     }
 }
 
+/// A pool for storing the memory used by [`UsageScope`]s and [`RenderBundleScope`]s. We take and store this memory when the
```

Comment:
Thanks for documenting!
If we decide to leave out render bundles we have to remove the mentions of it ofc.